doc/man/8/ceph.rst

```diff
@@ -385,13 +430,22 @@ Usage::
 
 	ceph fs rm <fs_name> {--yes-i-really-mean-it}
 
-Subcommand ``authorize`` creates a new client that will be authorized for the
-given path in ``<fs_name>``. Pass ``/`` to authorize for the entire FS.
-``<perms>`` below can be ``r``, ``rw`` or ``rwp``.
+Subcommand ``rm_data_pool``  removes the specified pool from FS's list of
+data pools. File data on this pool will become unavailable. Default data pool
+cannot be removed.
 
 Usage::
 
-    ceph fs authorize <fs_name> client.<client_id> <path> <perms> [<path> <perms>...]
+    ceph fs rm_data_pool <fs-name> <pool name/id>
+
+Subcommand ``set`` sets or updates a FS setting value for given FS name.
+
+Usage::
+
+    ceph fs set <fs-name> <fs-setting> <value>
+
+.. TODO:: add ``fs feature ls`` and ``fs flag set <flag-name> <flag-val>
```

Comment:
@vshankar @batrick Should these subcommands too be added to the man page?

Reply:
better place for `ceph tell` is https://docs.ceph.com/en/latest/cephfs/administration/

Reply:
`ceph tell` has numerous supported commands - better to include those in cephfs admin doc.

Reply:
@rishabh-d-dave We could link the admin doc from here instead?

Reply:
I suppose better to add it in Ceph man page because I think it's better to keep Ceph man page a standalone document that contains all necessary information offline.

Reply:
@vshankar Please reply to the first comment in this thread - https://github.com/ceph/ceph/pull/50646#discussion_r1146603870